packages/database/src/migrations/20210527034457-StriveLarvalHabitatBySpeciesBarChart-modifies-data.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+import { insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const DASHBOARD_GROUP_CODE = 'PG_Strive_PNG_Country';
+
+const SPECIES = [
+  'Ae. aegypti',
+  'Ae. albopictus',
+  'Ae. scutellaris',
+  'Aedes Other',
+  'Aedes spp.',
+  'An. farauti',
+  'An. koliensis',
+  'An. longirostris',
+  'An. punctulatus',
+  'Culex spp.',
+  'Cx. annulirostris',
+  'Cx. quinquefasciatus',
+  'Cx. sitiens',
+  'Mansoni spp.',
+];
+
+const COLOURS = [
+  '#e22709',
+  '#0df3e4',
+  '#fa6131',
+  '#9bbf87',
+  '#3fc399',
+  '#28d530',
+  '#4549a6',
+  '#c3d146',
+  '#67a6d1',
+  '#4e9469',
+  '#e617d4',
+  '#d84876',
+  '#4744cb',
+  '#b89fbd',
+];
+
+const BASE_DASHBOARD_REPORT = {
+  id: 'PG_Strive_Habitat_By_Species',
+  dataBuilder: 'reportServer',
+  dataServices: [
+    {
+      isDataRegional: true,
+    },
+  ],
+};
+
+const DATA_BUILDER_CONFIG = {
+  reportCode: 'STRIVE_Habitat_By_Species_LHS',
+};
+
+const getChartConfig = () => {
+  const chartConfig = {};
+  SPECIES.forEach((specie, index) => {
```

Comment:
```suggestion
  SPECIES.forEach((species, index) => {
```
Don't have to bother fixing this 😝

Reply:
I was feeling a bit weird to use `specie` but didn't really look into it.. Thanks my naming advisor @rohan-bes !